tests/views/unit.py

```diff
@@ -96,6 +96,43 @@ def test_submit_with_suggestion_and_comment(client, request_users, settings):
 
 
 @pytest.mark.django_db
+def test_submit_with_suggestion(client, request_users, settings):
+    """Tests translation can be applied after suggestion is accepted."""
+    settings.POOTLE_CAPTCHA_ENABLED = False
+    unit = Unit.objects.filter(suggestion__state='pending',
+                               state=UNTRANSLATED)[0]
+    sugg = Suggestion.objects.filter(unit=unit, state='pending')[0]
+    user = request_users["user"]
+    if user.username != "nobody":
+        client.login(
+            username=user.username,
+            password=request_users["password"])
+
+    url = '/xhr/units/%d/' % unit.id
+
+    response = client.post(
+        url,
+        {
+            'state': False,
+            'target_f_0': sugg.target_f,
+            'suggestion': sugg.id,
+        },
+        HTTP_X_REQUESTED_WITH='XMLHttpRequest'
+    )
+
+    if check_permission('translate', response.wsgi_request):
```

Comment:
The code in the test looks ok, but which specific user with translate right is used to test this? Or is this using `admin` user?